src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -5494,9 +5605,10 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
         UseHardlinksIfPossible="$(CreateHardLinksForPublishFilesIfPossible)"
         UseSymboliclinksIfPossible="$(CreateSymbolicLinksForPublishFilesIfPossible)"
-        Condition="'%(_DeploymentManifestDependencies.DependencyType)'=='Install'"/>
+        Condition="'$(PublishSingleFile)' != 'true' and '%(_DeploymentManifestDependencies.DependencyType)'=='Install'"/>
```

Comment:
I don't love that the single-file concept is required here when ideally it's an sdk concept. But I don't have a better idea.

Reply:
PublishSingleFile property is the only dependency I have with the dotnet SDK and I need that to filter file for the copy step. Agree it's not ideal to have such dependency from the SDK. If/lwhen we decide to drop legacy ClickOnce support, we could move this step into the SDK.